ironfish/src/rpc/routes/wallet/multisig/createSecret.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { ParticipantSecret } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace } from '../../namespaces'
+import { routes } from '../../router'
+import { AssertHasRpcContext } from '../../rpcContext'
+
+export type CreateSecretRequest = {
+  name: string
+}
+
+export type CreateSecretResponse = {
+  secret: string
+}
+export const CreateSecretRequestSchema: yup.ObjectSchema<CreateSecretRequest> = yup
+  .object({
+    name: yup.string().defined(),
+  })
+  .defined()
+
+export const CreateSecretResponseSchema: yup.ObjectSchema<CreateSecretResponse> = yup
+  .object({
+    secret: yup.string().defined(),
```

Comment:
Why returning the secrets? Ideally secrets (in any shape or form) should never escape the wallet. I'd return the public identity only

Reply:
I'll change to return only the identity. On that note it probably makes sense to rename to 'createIdentity'